en/releases/release-1.4.4.md

```diff
@@ -0,0 +1,29 @@
+---
+title: TiDB Operator 1.4.4 Release Notes
+---
+
+# TiDB Operator 1.4.4 Release Notes
+
+Release date: March 13, 2023
+
+TiDB Operator version: 1.4.4
+
+## New features
+
+- Support use volume-snapshot backup and restore when cluster has TiFlash ([#4812](https://github.com/pingcap/tidb-operator/pull/4812), [@fengou1](https://github.com/fengou1))
+
+- Support show a accurate backup size in volume-snapshot backup by calculate the backup size from snapshot storage usage ([#4819](https://github.com/pingcap/tidb-operator/pull/4819), [@fengou1](https://github.com/fengou1))
+
+- Support retry snapshot backup when its job or pod unexpected failure caused by kubernetes ([#4895](https://github.com/pingcap/tidb-operator/pull/4895), [@WizardXiao](https://github.com/WizardXiao))
+
+- Support integrated management log backup and snapshot backup in backup schedule ([#4904](https://github.com/pingcap/tidb-operator/pull/4904), [@WizardXiao](https://github.com/WizardXiao))
```

Comment:
```suggestion
- Support using volume-snapshot backup and restore on a TiDB cluster with TiFlash ([#4812](https://github.com/pingcap/tidb-operator/pull/4812), [@fengou1](https://github.com/fengou1))

- Show an accurate backup size in the volume-snapshot backup by calculating the backup size from snapshot storage usage ([#4819](https://github.com/pingcap/tidb-operator/pull/4819), [@fengou1](https://github.com/fengou1))

- Support retries for snapshot backups in case of unexpected failures caused by Kubernetes job or pod issues ([#4895](https://github.com/pingcap/tidb-operator/pull/4895), [@WizardXiao](https://github.com/WizardXiao))

- Support integrated management of log backup and snapshot backup in the `BackupSchedule` CR ([#4904](https://github.com/pingcap/tidb-operator/pull/4904), [@WizardXiao](https://github.com/WizardXiao))
```